client/src/components/Carousel.jsx

```diff
@@ -1,13 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Card from './Card.jsx';
+import axios from 'axios';
+
+const Carousel = ({ productId, cardType }) => {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    // if cardType is related
+    axios.get('/products/22122/related')
+      .then(response => {
+        console.log(response.data);
+        setProducts(response.data);
+      })
+      .catch(err => console.log(err));
+    // if cardType is outfit
```

Comment:
I'm not sure what this means? if the card is an outfit, it will error out on a GET request for related products?

Reply:
Ok, that makes sense.